usecase/pokemon.go

```diff
@@ -129,3 +131,81 @@ func (u PokemonUsecase) DeletePokemon(id string) (model.Pokemon, error) {
 
 	return pokemon, err
 }
+
+func (u PokemonUsecase) GetConcurrentPokemonCSV(types string, items string, itemsPerWorker string) (model.PokemonList, error) {
+	var pokemonList model.PokemonList
+	var err error
+
+	itemsInt, _ := strconv.Atoi(items)
+	itemsPerWorkerInt, err := strconv.Atoi(itemsPerWorker)
+	poolSize := utils.GetIntFromDivision(itemsInt, itemsPerWorkerInt)
+	limit := utils.GetIntFromDivision(len(pokemonList), poolSize)
+
+	fmt.Println(poolSize)
+	fmt.Println(limit)
+
+	if err == nil {
```

Comment:
You can avoid this indentation if you do 
```
if err != nil{
    err = errors.New("cannot get Pokemons")
    return pokemonList, err
}
```

<your logic here>

Reply:
👌